pattern-library/sass/patterns/_layouts.scss

```diff
@@ -212,216 +138,203 @@ $susy-view-reversed:    layout($susy-default auto $grid-direction-reversed); //
             .layout-col-a {
-                @include span(4 of 12 first);
+                @include span(4);
             }
 
             .layout-col-b {
-                @include span(8 of 12 last);
+                @include span(8);
             }
         }
     }
 }
 
 // 1/2 + 1/2 Layout
 .layout-1h1h {
-
     // CASE: standard layout
     &.layout {
-        @extend %layout-default;
-        @include susy-breakpoint($bp-screen-md, $susy-default) {
+        @include grid-container;
+        @media(min-width: $bp-screen-md) {
 
             .layout-col-a {
-                @include span(6 of 12 first);
+                @include span(6);
             }
 
             .layout-col-b {
-                @include span(6 of 12 last);
+                @include span(6);
             }
         }
     }
 
     // CASE: flush layout
     &.layout-flush {
-        @extend %layout-flush;
-        @include susy-breakpoint($bp-screen-md, $susy-flush) {
-
+        @include grid-container;
+        @media(min-width: $bp-screen-md) {
             .layout-col-a {
-                @include span(6 of 12 first);
+                @include span(6, before);
             }
 
             .layout-col-b {
-                @include span(6 of 12 last);
+                @include span(6, after);
             }
         }
     }
 
     // CASE: reversed layout
     &.layout-reversed {
-        @extend %layout-reversed;
-        @include susy-breakpoint($bp-screen-md, $susy-reversed) {
-
+        @include grid-container(row-reverse);
+        @media(min-width: $bp-screen-md) {
             .layout-col-a {
-                @include span(6 of 12 first);
+                @include span(6);
             }
 
             .layout-col-b {
-                @include span(6 of 12 last);
+                @include span(6);
             }
         }
     }
 }
 
 // 1/4 + 1/2 + 1/4 Layout
 .layout-1q1h1q {
-
     // CASE: standard layout
     &.layout {
-        @extend %layout-default;
-        @include susy-breakpoint($bp-screen-md, $susy-default) {
+        @include grid-container;
+        @media(min-width: $bp-screen-md) {
 
             .layout-col-a {
-                @include span(3 of 12 first);
+                @include span(3);
             }
 
             .layout-col-b {
-                @include span(6 of 12);
+                @include span(6);
             }
 
             .layout-col-c {
-                @include span(3 of 12 last);
+                @include span(3);
             }
         }
     }
 
     // CASE: flush layout
     &.layout-flush {
-        @extend %layout-flush;
-        @include susy-breakpoint($bp-screen-md, $susy-flush) {
-
+        @include grid-container;
+        @media(min-width: $bp-screen-md) {
             .layout-col-a {
-                @include span(3 of 12 first);
+                @include span(3, before);
             }
 
             .layout-col-b {
-                @include span(6 of 12);
+                @include span(6, none);
```

Comment:
I'm not so sure that my previous review comment of making 12 a variable makes sense, in that you'd then to do math in all these situations. Maybe it is simpler to be hard-coded for now.